.github/workflows/contributor-twitter.yml

```
@@ -0,0 +1,21 @@
+name: contributor-twitter
+on:
+  issues:
+    types: [labeled]
+jobs:
+  tweet:
+    runs-on: ubuntu-latest
+    steps:
+      # This has essentially zero costs and helps debugging if the workflow fails.
+      - uses: hmarr/debug-action@master
+      - uses: ethomson/send-tweet-action@v1
+        if: ${{ github.event.label.name == 'good first issue' && github.event.issue.state == 'open' }}
+        with:
+          status: >
+            A new issue on the @materialui repository has been marked as a "good first issue". Any help resolving this issue would be greatly appreciated. 
```

```suggestion
            A new issue in the @materialui repository has been labelled as a "good first issue". If you haven't contributed before, now's your chance!
```

---

Reading it again I don't think we need the quotes, as it still makes sense as a sentence:
```suggestion
            A new issue in the @materialui repository has been labelled as a good first issue. If you haven't contributed before, now's your chance!
```